boards/risc-v/mpfs/icicle/include/board.h

```diff
@@ -37,6 +37,10 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
+#if defined(CONFIG_MMCSD_MMCSUPPORT) && defined(CONFIG_MPFS_EMMCSD)
+#define MPFS_EMMC_CLK_200MHZ
```

Comment:
```suggestion
#  define MPFS_EMMC_CLK_MODE MPFS_EMMCSD_MODE_HS200
```
and simplify code in `mpfs_device_reset` by`priv->bus_speed = MPFS_EMMC_CLK_MODE;`?
```
priv->bus_speed = MPFS_EMMC_CLK_MODE
```

Reply:
Right!